modules/globebrowsing/shaders/rings_geom_fs.glsl

```diff
@@ -39,6 +40,11 @@ Fragment getFragment() {
   // The length of the texture coordinates vector is our distance from the center
   float radius = length(st);
 
+  // Discard if normal is perpendicular to the camera direction
+  if (abs(dot(normalize(vs_normal), vec3(0.0, 0.0, 1.0))) <= 1.0e-3) {
```

Comment:
```suggestion
  if (abs(dot(normalize(vs_normal), vec3(0.0, 0.0, 1.0))) <= 0.01) {
```

Changing it to 1e-2 to prevent more issues (for example 2025 APR-20 22:00:00
![image](https://github.com/user-attachments/assets/eca850a3-436b-41ab-9fb9-d73e5c65c8d1)


Changing it to `0.01` to make it easier to read